include/okapi/api/chassis/model/xDriveModel.hpp

```diff
@@ -124,6 +125,21 @@ class XDriveModel : public ChassisModel {
   virtual void
   xArcade(double irightSpeed, double iforwardSpeed, double iyaw, double ithreshold = 0);
 
+  /**
+   * Drive the robot with a field-oriented arcade drive layout. Uses voltage mode.
+   * 
+   * @param ixSpeed speed to the right
+   * @param iforwardSpeed speed forward
+   * @param iyaw turn speed
+   * @param iangle current chassis angle
```

Comment:
Can you also please document the winding direction of this angle? You just need to state that zero degrees applies no correction and that it winds clockwise.

Reply:
Yep, done!